### PR TITLE
[bitnami/ejbca] Add ejbca tests

### PR DESCRIPTION
**Pull request by joancafom**
### Description of the change

The main objective of this PR is to publish our Bitnami EJBCA Helm Chart using VMware Image Builder. In order to do that, several changes are included:

- Adding the asset to the publishing workflow
- Increasing the existing test coverage of the asset by adding Cypress tests.
- Increasing the existing test coverage of the asset by adding GOSS tests.

### Benefits

- Ensuring higher quality of the Helm charts.
- Increased pool of assets completely handled by VMware Image Builder.

### Possible drawbacks

Automated tests could show to be potentially flaky. 

### Applicable issues

NA

### Additional information

Tested in https://github.com/joancafom/charts/pull/85

### Checklist

- [x] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: .vib/ejbca/cypress/cypress/plugins/index.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+/// <reference types="cypress" />
```
Comment (FraPazGal) [inline .vib/ejbca/cypress/cypress/plugins/index.js:1]: I think you can remove this file.

File: .vib/ejbca/cypress/cypress/integration/ejbca_spec.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  const certFile = `cypress/downloads/${Cypress.env('username')}.p12`;
+  cy.task('fileExists', certFile).then((exists) => {
+    // A user can only enrol once
+    if (!exists) {
+      cy.visit('/ejbca/enrol/keystore.jsp');
+      cy.get('#textfieldusername').type(Cypress.env('username'));
+      cy.get('#textfieldpassword').type(Cypress.env('password'));
+      cy.get('#buttonsubmitusername').click();
+      cy.get('#tokenKeySpec').select('RSA_2048');
+      cy.get('#certprofile').select('ENDUSER');
```
Comment (FraPazGal) [inline .vib/ejbca/cypress/cypress/integration/ejbca_spec.js:13]: I would parametrize these constants, be it by using a fixtures file or by using a `const` variable. IIRC, we usually opt for the fixtures solution.

File: .vib/ejbca/goss/vars.yaml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+ejbcaCA:
+  name: ManagementTestCA
+  baseDN: "UID=c-1CCXmPAsNWmZuDtQQ8FHl7tcVdjCiNTH,O=ExampleCA,C=SE"
+podSecurityContext:
+  fsGroup: 1002
+containerSecurityContext:
+  runAsUser: 1002
+containerPorts:
+  http: 8081
+  https: 8443
```
Comment (FraPazGal) [inline .vib/ejbca/goss/vars.yaml:10]: Have you encountered issues changing the default value for `containerPorts.https`? It seems strange the `http` one can be changed but not the other one.

**Review by FraPazGal — CHANGES_REQUESTED** [review]
Thanks for the changes @joancafom! Please check my comments whenever possible.

File: .vib/ejbca/cypress/cypress/plugins/index.js (hunk 1 above)
Comment (joancafom) [reply .vib/ejbca/cypress/cypress/plugins/index.js:1]: No, I cannot as I am defining a custom `task` below. see line 21

File: .vib/ejbca/goss/vars.yaml (hunk 3 above)
Comment (joancafom) [reply .vib/ejbca/goss/vars.yaml:10]: Yes, changes in the values for this port do not actually change the port in the application (unlike the `http` one)

File: .vib/ejbca/cypress/cypress/plugins/index.js (hunk 1 above)
Comment (FraPazGal) [reply .vib/ejbca/cypress/cypress/plugins/index.js:1]: My bad then, forget I said anything 😁

**Review by FraPazGal — APPROVED** [review]
LGTM!

**Pull request merged**